docs/development.md

```diff
@@ -1,54 +1,113 @@
 # Oak Development
 
+- [Meta-Advice](#meta-advice)
+- [Prerequisites](#prerequisites)
+- [Run Example Application](#run-example-application)
+  - [Build Application](#build-application)
+  - [Build Runtime Server](#build-runtime-server)
+  - [Run Runtime Server](#run-runtime-server)
+  - [Run Example Client](#run-example-client)
+
+## Meta-Advice
+
+For any open-source project, the best way of figuring out what prerequisites and
+dependendencies are required is to look at the projects continuous integration
+(CI) scripts. A project's top-level [Travis](https://travis-ci.org/),
+[CircleCI](https://circleci.com/),
+[Cloudbuild](https://cloud.google.com/cloud-build) or
+[Dockerfile](https://docs.docker.com/) will give definitive information about
+what's needed, and at what version.
```

Comment:
I'm not entirely certain I understand why we need this paragraph. I think for this doc, the objective for most reader's is retrieving the minimum information that's required to get started. 

Given that this is more general context on starting with unknown projects, perhaps this would be better placed somewhere else?

Reply:
I also find this a bit awkward, especially since we don't actually use Travis or CircleCI, so a careless reader skimming through this doc may be tempted to just assume we do, and be confused by this.

Reply:
Dropped mentions of other CI systems.

Reply:
I like the intend of "if you're stuck, looking at the CI might help", but perhaps making it more specific to our project could be cool.